Simplified.xcodeproj/project.pbxproj

```diff
@@ -6551,6 +6566,7 @@
 				CODE_SIGN_ENTITLEMENTS = Simplified/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CURRENT_PROJECT_VERSION = 7;
+				CURRENT_PROJECT_VERSION = 6;
```

Comment:
I think Xcode messed up here :) can you remove one of these?

Reply:
Fixed